keras_cv/layers/preprocessing/grid_mask.py

```diff
@@ -213,7 +213,7 @@ def _augment_images(self, images):
             images = tf.expand_dims(images, axis=0)
 
         # TODO: Make the batch operation vectorize.
-        output = tf.map_fn(lambda image: self._grid_mask(image), images)
+        output = tf.vectorized_map(lambda image: self._grid_mask(image), images, True)
```

Comment:
Thanks for the PR.  Please add fallback_to_while_loop=True for readability instead of just `True`.  

thanks!